osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -265,6 +276,18 @@ protected virtual void BindProcessor(ScoreProcessor processor)
             ComboCounter?.Current.BindTo(processor.Combo);
             HealthDisplay?.Current.BindTo(processor.Health);
 
+            if (LeftHitErrorDisplay != null)
+            {
+                processor.NewJudgement += LeftHitErrorDisplay.OnNewJudgement;
+                LeftHitErrorDisplay.HitWindows = processor.CreateHitWindows();
+            }
+
+            if (RightHitErrorDisplay != null)
```

Comment:
this is kinda ugly, not sure if it's better but you can iterate over visibility container and find hit error displays and bind the events and hit windows